source/reduced.js

```diff
@@ -7,7 +7,7 @@ import _reduced from './internal/_reduced';
  * box: the internal structure is not guaranteed to be stable.
  *
  * Note: this optimization is only available to the below functions:
```

Comment:
This note should flat out say that this optimization is only for transducers.

Reply:
That'd conflict with #2663.